packages/DataGrid/src/components/RowIndicator/RowIndicator.styles.js

```diff
@@ -0,0 +1,48 @@
+import styled from "styled-components";
+import tokens from "@paprika/tokens";
+
+export const RowContainer = styled.div`
+  align-items: center;
+  cursor: pointer;
+  display: flex;
+  height: 100%;
+  justify-content: flex-start;
+  padding: 0;
+  width: 100%;
+`;
+
+export const RowIndexText = styled.div.attrs(({ hasFourDigitsOrMore }) => {
+  const style = hasFourDigitsOrMore
+    ? {
+        fontSize: "11px",
+        padding: "4px",
+      }
+    : {
+        fontSize: "13px",
+        padding: "6px",
+      };
```

Comment:
tokens

Reply:
same here I don't think we have 6px and 13px and not sure 100% the UI for this part, we will polish it later, maybe whoever is working on the feature needing this can help. or myself in my next round